crates/stdx/src/lib.rs

```diff
@@ -5,7 +5,7 @@ use std::{cell::Cell, fmt};
 /// Appends formatted string to a `String`.
 #[macro_export]
 macro_rules! format_to {
-    (&buf:expr) => ();
+    ($buf:expr) => ();
```

Comment:
@matklad , is this overload required at all?

Reply:
just for completeness, but yeah, it maybe makes sense to remove it